mrjob/dataproc.py

```diff
@@ -1130,16 +1123,14 @@ def _cluster_create_kwargs(self):
             gcs_init_script_uris.append(
                 self._upload_mgr.uri(self._master_bootstrap_script_path))
 
-            # always add idle termination script
-            # add it last, so that we don't count bootstrapping as idle time
-            gcs_init_script_uris.append(
-                self._upload_mgr.uri(_MAX_MINS_IDLE_BOOTSTRAP_ACTION_PATH))
-
         # NOTE - Cluster initialization_actions can only take scripts with no
         # script args, so the auto-term script receives 'mrjob-max-secs-idle'
         # via metadata instead of as an arg
         cluster_metadata = dict()
         cluster_metadata['mrjob-version'] = mrjob.__version__
+
+        # TODO: remove this once lifecycle_config is visible through
```

Comment:
Remove?

lifecycle_config referenced on line 1196

Reply:
This is specifically for the `mrjob-max-secs-idle` metadata.

We don't need this anymore for the initialization action script that handles idle timeout, but it's also currently the only way you can see idle timeout information for your job in the Google Cloud Console or through the `gcloud` util, so I thought it was worth leaving in for now. Once `lifecycle_config` becomes generally visible, we can get rid of the metadata variable.

Reply:
Oh, the NOTE above this is stale though. Removed in my release branch. Thanks!